docs/CONTRIBUTING.md

```diff
@@ -4,6 +4,8 @@ Hi, glad you are here all contributions are very much welcome!
 
 Before you start Contributing to Canopy it is worth familiarising yourself with the [best practice](./BEST_PRACTICE.md) guidelines.
 
+Please do raise an [issue](https://github.com/Legal-and-General/canopy/issues) or [discussion](https://github.com/Legal-and-General/canopy/discussions) before starting a piece of work. Our contributors can help provide guidance to make sure your effort isn't wasted. It is important that the development and design implementations stay in sync and that the codebase remains consistent. This will help to give the codebase a long life and remain intuitive for those using it.
```

Comment:
Maybe also mention the Slack channel? Was also thinking that we could mention how important the review process is, as these components will end up in the hands of many developer over many apps. May help with the feeling that some new devs feel that the PR process can be longer and more in depth than they're used to.